src/Resources/skeleton/event/Subscriber.tpl.php

```diff
@@ -15,7 +15,7 @@ public function <?= $method_name ?>(<?= $event_arg ?>)
     public static function getSubscribedEvents()
     {
         return [
-            '<?= $event ?>' => '<?= $method_name ?>',
+           <?= $event ?> => '<?= $method_name ?>',
```

Comment:
If $event is a class name, we'll also need a use statement for it.

Reply:
The variable $event_full_class_name will have the full class name so a use statement will be inserted on the line 6.

Reply:
Of course :)